src/pages/staticPages/tools/mutationMapper/MutationMapperToolStore.ts

```diff
@@ -168,7 +168,7 @@ export default class MutationMapperToolStore
                     const getMutations = () => {
                         return this.mutationsByGene[gene.hugoGeneSymbol];
                     };
-                    map[gene.hugoGeneSymbol] = new MutationMapperStore(Object.assign({}, AppConfig, {filterMutationsBySelectedTranscript:!this.hasInputWithProteinChanges}),
+                    map[gene.hugoGeneSymbol] = new MutationMapperStore(AppConfig.serverConfig, {filterMutationsBySelectedTranscript:!this.hasInputWithProteinChanges},
```

Comment:
this is following up on a TODO that I think Onur put in to separate true app configuration from what i think is some kind of run time detection (hasInput ...)   
@inodb @onursumer let me know if you object to the very generic AppConfig being referenced deeply inside this MutationMapper code.

Reply:
yeah i put that comment - thanks for separating it out, this is cleaner